

// find error in the code
fn plus_five(value: i32) -> isize {
    value + 1
}

fn main() {
    let number = plus_five(5);

    println!("The value of x is:{}", number);
}
